pkg/metrics/metrics.go

```diff
@@ -129,6 +129,36 @@ var (
 		Help:        "Last timestamp when we received an event",
 		ConstLabels: prometheus.Labels{"source": LabelEventSourceAPI},
 	})
+
+	// Proxy statistics
+
+	// ProxyParseErrors is a count of failed parse errors on proxy
+	ProxyParseErrors = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Name:      "proxy_parse_errors_total",
```

Comment:
I recommend to not call these proxy_. Users will not know what it means. We should also make sure that it is clear that these are relevant when policy enforcement is active.

policy_l7_parse_error_total
policy_l7_forwarded_total
policy_l7_denied_total
policy_l7_processed_total

Reply:
sure.

Reply:
done